src/volur/sdk/v1alpha2/sources/csv/base.py

```diff
@@ -284,28 +314,60 @@ def get_value(
 
 
 @dataclass
-class ProductsSource:
-    """
-    Base class for the product sources.
-    This class in an abstract class that defines the interface for the products CSV
-    source.
-    """
+class CharacteristicColumnDate(CharacteristicColumn):
+    date_formats: list[str] = field(init=False)
+    default_date_format: list[str] = field(
+        default_factory=lambda: ["%d-%m-%Y"], init=False
+    )
+    extra_date_formats: list[str] = field(default_factory=list)
```

Comment:
We actually can get rid of these lines https://github.com/volur-ai/python-volur-sdk/pull/64/files#diff-ebc81713c7c4d4abf67b22af35f751483070e3b8c7c569eb131654a39e9ad78cR332-R335 below

```suggestion
    date_formats: list[str] = field(init=False)
    default_date_format: list[str] = field(
        default_factory=lambda: [
          "%d-%m-%Y",
          "%Y-%m-%d",
          "%Y/%m/%d",
          "%d-%m-%Y",
          "%d/%m/%Y",
        ], init=False
    )
    extra_date_formats: list[str] = field(default_factory=list)
```